simperby/src/types.rs

```diff
@@ -43,14 +43,12 @@ pub enum CommitInfo {
 
 /// A configuration for a node.
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct Config {
-    /// TODO: remove this and introduce a proper peer discovery protocol
-    pub peers: Vec<Peer>,
-}
+pub struct Config {}
```

Comment:
Now then, what does this `Config` exist for?

Reply:
Think it as a placeholder. If we add `Config` later by altering the interface of construction methods, it will breakup the code in a quite ugly way (whereas adding fields in the existing placeholder is much easier)